std/json.d

```diff
@@ -969,10 +969,19 @@ if(isInputRange!T)
         depth--;
     }
 
-    parseValue(&root);
+    parseValue(root);
     return root;
 }
 
+version(unittest)
```

Comment:
From 2.068.0,  `unittest` block is not parsed unless it is compiled with `-unittest` option.
Now using `unittest` block is better.
I will replace it.